AppRun

```diff
@@ -0,0 +1,58 @@
+#! /usr/bin/env bash
+
+
+# make sure errors in sourced scripts will cause this script to stop
+set -e
+
+this_dir="$(readlink -f "$(dirname "$0")")"
+bin_dir="/usr/bin"
+
+source "$this_dir"/apprun-hooks/"linuxdeploy-plugin-qt-hook.sh"
+
+my_help() {
+    echo "Usage (Meta):"
+    echo
+    echo "To run a specific sub-programm just call the AppImage"
+    echo "with the program name as the first parameter."
+    echo
+    echo "Available programs:"
+    echo '"tiled" (the main program. Default)'
+    echo '"terraingenerator"'
+    echo '"tmxrasterizer" (convert projects to images)'
+    echo '"tmxviewer" (view your projects)'
+    echo
+    echo "Example:"
+    echo "${0} tmxrasterizer --ignore-visibility"
```

Comment:
Unfortunately this isn't really helpful since it results in:
```
Example:
/tmp/.mount_Tiled-PXnnDY/AppRun.wrapped tmxrasterizer --ignore-visibility
```
I'll just keep it for now, but maybe you have an idea to improve this?